site/_posts/2021-02-19-octant-0.17-release.md

```diff
@@ -0,0 +1,74 @@
+categories: ['kubernetes']
+tags: ['octant', 'release']
+---
+
+<h1>Kubernetes: there is an app for that</h1>
+
+Turning Octant into a desktop application has long been a goal of ours and we are pleased to announce that with the release of 0.17 we are shipping a desktop application for all three major operating systems. The command-line version will still be available for the next couple of releases to allow a smooth transition for everyone,after which we will be 100% focused on the desktop version.
+
+![Octant Electron applications view](/img/posts/2021/02/19/electron-applications.png)
+
+
+To create a desktop version of Octant we used Electron. Electron is a popular cross-platform framework used by many well known desktop applications. Electron is a mature technology built on top of Chromium engine with a solid production environment and great community support. Leveling up to a desktop application opens the doors for many new and exciting things Octant can leverage in the future to improve the overall user experience.
+
+Accessing kubernetes clusters from an application brings many benefits, especially access to the hardware and operating system components. One of the benefits we took advantage right away is access to the System tray. Octant can run in the background and be accessible with just a single click from the System tray. If you want to see the logs, you can get to them right there from the System tray. Expect more actions available from the System tray in the near future.
+
+![Octant Electron System Tray](/img/posts/2021/02/19/electron-system-tray.png)
+
+The header also went through a few changes. Since we are not operating inside the browser anymore, we had to add the Forward/Back buttons to the header. Additionally, header responsiveness has been improved to make it more usable at lower resolutions. 
+
+A thing to note: In this release macOS needs an additional step when running Octant for the very first time. You will have to locate Octant in Finder, control-click the Octant icon, then choose Open. More details about running unsigned apps on Mac are available [here](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac). This will be resolved in future releases.
+
+<h3>Anatomy of the Navigation</h3>
+
+
+Based on user feedback, we decided to redesign the Octant navigation in order to accommodate the growing number of Octant plugins.  The left navigation is now separated into two vertical panels: a left panel with a list of all known modules and each module has its own panel to the right. Module panels contain all module specific navigation actions as well as module description at the bottom. The module description section right now displays just the description text, we anticipate opening that up in the future to allow inclusion of more interactive UI elements. This is a great news for Octant plugin authors, giving their plugin better visibility and discoverability. 
```

Comment:
Remove: `right now`

Awkward phrasing in last sentence: maybe something like "Plugin authors can use these features to improve plugin visibility and discoverability."